src/Vlingo.Common/Compiler/DynaCompiler.cs

```diff
@@ -35,12 +35,16 @@ public Type Compile(Input input)
                 {
                     typeof(object).GetTypeInfo().Assembly,
                     input.Protocol.GetTypeInfo().Assembly,
+                    GetType().Assembly, // Adding Vlingo.Common
                     Assembly.GetCallingAssembly(),
                     Assembly.Load(new AssemblyName("System.Runtime")),
                     Assembly.Load(new AssemblyName("mscorlib")),
                     Assembly.Load(new AssemblyName("netstandard")),
                 };
 
+                // Adding Vlingo.Actors, if available
+                TryLoadActorsAssembly(assembliesToLoad);
```

Comment:
I would bet that GetReferencedAssemblies() loaded actors dépendance every time. I’m surprised it’s not the case as it should be referenced almost everywhere except the common